### PR TITLE
Update restioson.toml

### PR DESCRIPTION
**Pull request by Restioson**

### CODE REVIEW

**Comment by Restioson** [comment]
:thinking: bio could do with something more, taking suggestions

File: data/people/restioson.toml [hunk 1]
```diff
@@ -7,5 +7,11 @@ groups = [
     "building",
 ]
 
+pronouns = "he/him"
+
+bio = "Help, I'm trapped inside a website!"
```
Comment (Restioson) [inline data/people/restioson.toml]:
```suggestion
bio = "Help, I'm trapped inside a website! Trans rights."
```

**Review by ashhhleyyy — APPROVED** [review]
I should probably fix the dodgy actions workflow that's built and pushed this to the other branch already :smile:

**Comment by Restioson** [comment]
Can I merge then?

**Comment by ashhhleyyy** [comment]
Yep!

**Pull request merged**